pkg/pkg.ml

```diff
@@ -3,13 +3,27 @@
 #require "topkg"
 open Topkg
 
+let includes = function
+  | "prometheus" | "prometheus-app" -> ["prometheus"]
+  | "datakit-ci" -> ["ci"]
+  | "datakit" -> ["src"; "src/datakit"]
+  | "datakit-server" -> ["src"; "src/datakit-server"]
+  | "datakit-github" -> ["src"; "src/datakit"]
+  | "datakit-client" -> ["src"; "src/datakit-client"]
+  | x -> failwith ("Unknown includes for package: " ^ x)
```

Comment:
@samoht: Note: I've changed the build so that each package only sees its own files. Hopefully this should be more reliable, and we can have more than one file called `main.ml`...